frontend/package.json

```diff
@@ -56,6 +56,7 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-string-helpers": "^2.0.0",
+    "ember-cli-template-lint": "^1.0.0",
```

Comment:
@godfreyyeung is this the add-on that connects the template lint rules to ember tests?

Reply:
Asking just because I didn't see any explicit tests anywhere, wondering if this is part of ember magic

Reply:
yep! It will automatically add the linting tests